### PR TITLE
form and functionality added

### PR DESCRIPTION
**Pull request by lixodeika67**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -33,4 +33,50 @@ for (let i=0; i < skills.length; i++){
...
+
+    const usersName = event.target.usersName.value;
+    const usersEmail = event.target.usersEmail.value;
+    const usersMessage = event.target.usersMessage.value;
+
+    console.log('Name:', usersName);
+    console.log('Email:', usersEmail);
+    console.log('Message:', usersMessage);
+
+    ////
```
Comment (josiahandrew12) [inline js/index.js:52]: What is this for?

File: js/index.js [hunk 2]
```diff
@@ -33,4 +33,50 @@ for (let i=0; i < skills.length; i++){
...
+
+     const entry = removeButton.parentNode;
+     entry.remove();
+ });
+
+
+ newMessage.appendChild(removeButton);
+ messageList.appendChild(newMessage);
+
+    ////
```
Comment (josiahandrew12) [inline js/index.js:78]: What is this for?

File: index.html [hunk 3]
```diff
@@ -82,6 +82,29 @@ <h2>Connect</h2>
         fa-github"></i></a>
     
 </section>
+<section id="LeaveMessage">
```
Comment (josiahandrew12) [inline index.html:85]: 👍

File: js/index.js (hunk 1 above)
Comment (lixodeika67) [reply js/index.js:52]: Just for me. to see if it is working.

File: js/index.js (hunk 2 above)
Comment (lixodeika67) [reply js/index.js:78]: Just to see the message on the page and be able to remove it. I will fix it for the final project

**Pull request merged**